packages/cli/package.json

```diff
@@ -30,7 +30,7 @@
     "start:default": "cd bin && ./n8n",
     "start:windows": "cd bin && n8n",
     "test": "npm run test:sqlite",
-    "test:sqlite": "export DB_TYPE=sqlite && jest --forceExit",
+    "test:sqlite": "export N8N_LOG_OUTPUT=file; export DB_TYPE=sqlite; jest",
```

Comment:
`export N8N_LOG_OUTPUT=file` to declutter console output from expected errors by default.

Reply:
do we then always have to remove a file?

Reply:
Solved more cleanly by adding a new `silent` log level.